view/player/tmpl/default.html.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Nooku Framework - http://nooku.org/framework
+ *
+ * @copyright	Copyright (C) 2011 - 2014 Johan Janssens and Timble CVBA. (http://www.timble.net)
```

Comment:
It's oki. This is in com:files. Need to change all the headers here. @ercanozkaya We need to make tickets for this across all repo's and update headers. Remove Nooku references update dates etc.